src/app/shared/services/records/record.service.ts

```diff
@@ -154,4 +154,17 @@ export class RecordService extends GenericListService {
     data.set('content', feedbackData);
     return this.http.post(`${environment.API_URL}/campus-event-feedbacks/`, data);
   }
+
+  updateRecordStatus(recordId: number, isApproved: boolean, isDepartmentAdmin: boolean) {
+    if (isDepartmentAdmin) {
+      return this.http.post(`${environment.API_URL}/records/${recordId}/department-admin-review/`, {});
+    }
+    return this.http.post(`${environment.API_URL}/records/${recordId}/school-admin-review/`, {});
+  }
+
+  closeRecord(recordId: number, isSchoolAdmin: boolean) {
+    if (isSchoolAdmin) {
```

Comment:
No need to check role type, our backend will raise error if user has no permission.